StatEngine/stat-engine
# Correct default behavior on simultaneous-incidents dashboard for chlo…

…ropleth maps

## Overview
Fix the default behavior of the chloropleth maps in the simultaneous incidents dashboard.

## GitHub Issues
-

## Changes
-

## Screenshots / Videos


## Steps to Test
- run [simultaneous-incidents](https://github.com/ProminentEdge/se-simultaneous-incidents) for 2 departments far enough to get some simultaneous incidents data.
- load StatEngine
- create workspace, include simultaneous-incidents among the default dashboards
- open simultaneous incidents
- Examine Chloropleth maps titled "Response times by census tract" and "Maximum number of in-progress incidents per tract" Both should display colored census tracts over a base map with data on hover and build filters when clicked upon.
